src/middleware.js

```diff
@@ -14,8 +14,11 @@ export default ({ dispatch }) => next => action => {
       )
       break
     case `user_${dataFetchActionTypes.FETCH_SUCCESS}`:
+      const { planName } = getState().productFeatures
```

Comment:
you could use `const { productFeatures: { planName } } = getState()` here, just fyi
this is fine though, just thought i'd mention it :)